### PR TITLE
The Great Fishing Chance Rework [DONE?]

### PR DESCRIPTION
**Pull request by InsightfulParasite**
## About The Pull Request
Reworking fishing chance on previous complaints about the chances being too large or too small.

- Changes to how fishing chance is calculated. Now a rod has a fishing chance modifier that is improved additivly by hooks and reel. 0.6 basic rod + basic hook 0.2 + basic line 0.2 = 1 chance modifier. 
- The formula for a rare fish is (15 - (5 * [amount of times we succeeded])) * [chance modifier] Basically there is a 85% chance for basic and a 15% chance for uncommon. If you roll uncommon it will roll again to see if you get rare which would be 1.5%.
- Fishing delay has been reduced to around 8 seconds and fishing skill now reduces the time it takes to fish by a flat second based on level.
- Adds a book that grants the reader the first level in fishing. The fishing skill book is included in the fishing tacklebox from the fish_market.
- Dough, canned peaches, and  space cash has been removed from the saltwater and freshwater loot table.
- Common, Uncommon, and Rare loot have been put into separate loot tables that have item values that each add up to 100%.
- Small edit to shrimp to make them contain only 2 units of reagents.

## Why It's Good For The Game
Fixes up and improves a small part of the game.

## Changelog
:cl:
tweak: fishing chance.
tweak: shrimp reagents
/:cl:

### CODE REVIEW

**Comment by Litberries** [comment]
this is just a me thing but I feel like stats should be assigned to the brain like spells cus you lose all your stats on gibbing

**Comment by Litberries** [comment]
oh shoot wrong PR

File: ModularTegustation/fishing/code/rod_components/fishing_hooks.dm [hunk 1]
```diff
@@ -18,7 +18,7 @@
 	name = "bone hook"
 	desc = "a simple hook carved from sharpened bone"
 	icon_state = "bone"
-	fishing_value = 200
+	fishing_value = 0.1
```
Comment (LanceSmites328) [inline ModularTegustation/fishing/code/rod_components/fishing_hooks.dm:21]: This is now a downgrade (base is 0.2, reportedly)

File: ModularTegustation/fishing/code/rod_components/fishing_lines.dm [hunk 2]
```diff
@@ -17,7 +17,7 @@
 	name = "fishing sinew line"
 	desc = "An all-natural fishing line made of stretched out sinew."
 	icon_state = "sinew"
-	fishing_value = 200
+	fishing_value = 0.1
```
Comment (LanceSmites328) [inline ModularTegustation/fishing/code/rod_components/fishing_lines.dm:20]: See previous comment

**Review by LanceSmites328 — CHANGES_REQUESTED** [review]

File: ModularTegustation/fishing/code/rod_components/fishing_hooks.dm (hunk 1 above)
Comment (InsightfulParasite) [reply ModularTegustation/fishing/code/rod_components/fishing_hooks.dm:21]: changed it to 0.3

**Review by LanceSmites328 — APPROVED** [review]

**Pull request merged**